src/main/java/org/team1540/robot2018/motion/FollowProfile.java

```diff
@@ -0,0 +1,145 @@
+package org.team1540.robot2018.motion;
```

Comment:
Honestly can't properly review this without a lot more time and effort, so a lot of trust here. Also put this in ROOSTER at some point SMH.

Reply:
You should be able to trust it considering we ran it all of PNWs lol

And yes I will but that's a lot of effort

Reply:
Working code != working code